src/main/java/com/amihaiemil/docker/RemoteDocker.java

```diff
@@ -45,24 +46,34 @@
     /**
      * Remote Docker engine. API version is 1.35 by default.
      * @param uri Remote Docker URI.
-     * @param certs Path to the folder containing these 3:
-     *  CA certificate (ca.pem), client certificate (cert.pem)
-     *  and client key (key.pem).
+     * @param keys Path to the keystore.
+     * @param trust Path to the truststore.
+     * @param storePwd Password for the keystore.
+     * @param keyPwd Passphrase for the key.
      */
-    public RemoteDocker(final URI uri, final Path certs) {
-        this(uri, "v1.35", certs);
+    public RemoteDocker(
+        final URI uri, final Path keys, final Path trust,
+        final char[] storePwd, final char[] keyPwd) {
+        this(uri, "v1.35", keys, trust, storePwd, keyPwd);
     }
 
     /**
      * Remote Docker engine.
      * @param uri Remote Docker URI.
      * @param version API version (eg. v1.35).
-     * @param certs Path to the folder containing these 3:
-     *  CA certificate (ca.pem), client certificate (cert.pem)
-     *  and client key (key.pem).
+     * @param keys Path to the keystore.
+     * @param trust Path to the truststore.
+     * @param storePwd Password for the keystore.
+     * @param keyPwd Passphrase for the key.
      */
-    public RemoteDocker(final URI uri, final String version, final Path certs) {
-        this(new SslHttpClient(certs), uri, version);
+    public RemoteDocker(
```

Comment:
@llorllale Can you make this class package-protected (class and ctors) for now? It won't be visible/usable in version ``0.0.1`` -- which I would like to release this week, we already have LocalDocker + Containers and Images API.